### PR TITLE
chore(updatecli) finish cleaning up updatecli configuration from github to jenkins

### PR DESCRIPTION
**Pull request by dduportal**
Before this change:
```
updateJenkinsfile
-----------------

**Dry Run enabled**

✔ File content for "/var/folders/2s/09szbrgn22l2tcvxz1_k34g40000gn/T/updatecli/jenkins-infra/openvpn/Jenkinsfile_k8s", updated. 

--- /var/folders/2s/09szbrgn22l2tcvxz1_k34g40000gn/T/updatecli/jenkins-infra/openvpn/Jenkinsfile_k8s
+++ /var/folders/2s/09szbrgn22l2tcvxz1_k34g40000gn/T/updatecli/jenkins-infra/openvpn/Jenkinsfile_k8s
@@ -5,7 +5,7 @@
       containers: [
         containerTemplate(
           name: 'golang',
-          image: 'golang:1.17.3',
+          image: 'golang:1.17.3
           command: 'cat',
           ttyEnabled: true,
           resourceRequestCpu: '2',
           ```

After the change, no update :)

### CODE REVIEW

File: updatecli/updatecli.d/golang.yaml [hunk 1]
```diff
@@ -60,8 +60,10 @@ targets:
     kind: file
     spec:
       file: ./Jenkinsfile_k8s
-      matchPattern: 'golang:.*'
-      replacePattern: 'golang:{{ source `latestGoVersion` }}'
+      matchPattern: >-
+        'golang:.*'
+      replacePattern: >-
+        'golang:{{ source `latestGoVersion` }}'
```
Comment (lemeurherve) [inline updatecli/updatecli.d/golang.yaml:66]: Why with `>-` instead of the same line?

Comment (lemeurherve) [reply updatecli/updatecli.d/golang.yaml:66]: Was there a problem of regexp parsing?

Comment (dduportal) [reply updatecli/updatecli.d/golang.yaml:66]: yep, the single quote escaping broke my head, so I went to the block litteral.

Comment (dduportal) [reply updatecli/updatecli.d/golang.yaml:66]: (e.g. I'm lazy :D )

**Review by lemeurherve — APPROVED** [review]

**Pull request merged**